examples/kubernetes/templates/v1/cilium-cm.yaml

```diff
@@ -25,6 +25,8 @@ data:
   # If you want to run cilium in debug mode change this value to true
   debug: "false"
   disable-ipv4: "false"
+  # If you want to clean cilium state; uncomment the following lines
+  #clean-cilium-state: "true"
```

Comment:
You can uncomment the line and have the value set to `false`, no?

Reply:
I was just checking if this ENV variable is present.. I will change this